src/Controller/RecommendationsController.php

```diff
@@ -15,6 +15,9 @@
 
 class RecommendationsController extends BaseController
 {
+    const ITEMS_PAGE = 10;  // items on main page
```

Comment:
since php7.1 visibility modifier are available for PHP and recommended to use
```suggestion
    private const ITEMS_PAGE = 10;  // items on main page
```
Same can be done for ITEMS_BOX